test/i-frame.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { browser } from 'protractor';
+import {
+  IFramePage,
+} from '../src/page';
+
+describe('Given a practice iframe page', () => {
+  beforeAll(async () => {
+    browser.waitForAngularEnabled(false);
+    await browser.get('https://demoqa.com/frames');
+  });
+
+  describe('When the height of an iframe wants to be modify', () => {
+    const iFramePage: IFramePage = new IFramePage();
+    const height = 500;
+    beforeAll(async () => {
+      await iFramePage.setFormFrameHeight(height);
+    });
+
+    it('Then it should have a new height', async () => {
+      expect(await iFramePage.getIframeHeight()).toEqual(height);
```

Comment:
this is **optional** but for tests I prefer to keep these values directly here instead of assign them to the `height` variable... there are cases when variables make more sense, but if you read the test case like: 
```js
await iFramePage.setFormFrameHeight(500);
expect(await iFramePage.getIframeHeight()).toEqual(500);
```

you know exactly what is your test data despite having "duplicated" values, also, this makes sure what your input is independent of your expected output